iina/PlaylistViewController.swift

```diff
@@ -307,16 +307,17 @@ class PlaylistViewController: NSViewController, NSTableViewDataSource, NSTableVi
   // MARK: - IBActions
 
   @IBAction func addToPlaylistBtnAction(_ sender: AnyObject) {
-    Utility.quickOpenPanel(title: "Add to playlist", isDir: false) { (url) in
-      if url.isFileURL {
-        self.playerCore.addToPlaylist(url.path)
-        self.reloadData(playlist: true, chapters: false)
-        self.mainWindow.displayOSD(.addToPlaylist(1))
+    Utility.quickMultipleOpenPanel(title: "Add to playlist") { urls in
+      for url in urls {
+        if url.isFileURL {
+          self.playerCore.addToPlaylist(url.path)
+          self.mainWindow.displayOSD(.addToPlaylist(1))
```

Comment:
Please use `PlayerCore#sendOSD` to post OSD safely onto main thread. 

Also, why show OSD for each added file rather than for all of them? 

```swift

for url in urls {
  ...
}
playerCore.sendOSD(.addToPlaylist(urls.count))

```